accounts/urls.py

```diff
@@ -3,7 +3,7 @@
 from .views import UserViewSet
 
 router = DefaultRouter()
-router.register(r"users", UserViewSet)
+router.register(r"users", UserViewSet, basename="users")
```

Comment:
nit: we removed basenames in #57 since we're not rly using them – would be nice to be consistent! sorry for not communicating this earlier 😓